about.css

```diff
@@ -0,0 +1,189 @@
+@import url('http://fonts.googleapis.com/css?family=Lato:400,700');
+
+* {
+  font-family: 'Lato', sans-serif;
+  font-style: normal;
+}
```

Comment:
- [ ] Dear @Elhambasir, You did well using one of the main fonts for the design `Lato`. But please notice you have not used the other primary font to design your text. Please make use of the `Cocogoose` font in parts of the document which need a different layout.